Singleton/SceneInstance.cs

```diff
@@ -2,16 +2,16 @@
 
 public class SceneInstance<T> : MonoBehaviour where T : class, new() {
 	//
-	private static T _Instance = null;
+	protected static T _Instance = null;
```

Comment:
its so the inheriting class can modify the value if necessary, but external entities cannot.